app/routes/public/schedule/index.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const { Route, RSVP } = Ember;
+export default Route.extend({
+  titleToken() {
+    return this.l10n.t('Schedule');
+  },
+  templateName: 'public/schedule/room',
+  model() {
+    return RSVP.hash({
```

Comment:
Why a ```RSVP.hash```? You are returning a single model only. Please correct me if you are using it intentionally.

Reply:
@sumedh123 Earlier I was trying to get multiple models to display the schedule in rooms view. But since it has not been added I'll remove it.